app/controllers/books_controller.rb

```diff
@@ -0,0 +1,31 @@
+class BooksController < ApplicationController
+
+	def index
+		@books = Book.all
+    @title = "Lending Library"
+	end
+
+	def new
+		@book = Book.new
+	end
+
+	def create
+    #spam catch to redirect - maybe the bots were ignoring the spam field
+    redirect_to books_url and return if book_params[:username].present?
+    @book = Book.new(book_params)
+
+    respond_to do |format|
+      if @book.save
+        format.html { redirect_to books_path, notice: 'Thank you for listing your book.' }
+      else
+        format.html { render action: "new" }
+      end
+    end
+  end
+
+  private
+
+  def book_params
+    params.require(:book).permit(:title, :author, :name, :email, :comment, :username)
+  end
+end
```

Comment:
The whitespace is off in several files. It looks like you might be mixing tabs and spaces?